lighthouse-logger/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.1",
   "license": "Apache-2.0",
   "dependencies": {
-    "debug": "^2.6.8"
+    "debug": "^2.6.8",
+    "marky": "^1.2.0"
```

Comment:
lighthouse depends on lighthouse-logger explicitly so we don't need to update root package.json too?